bbp/modules/default.nix

```diff
@@ -682,22 +682,47 @@ let
             moduleFilePrefix = "nix/nse";
 
             description = "neuroM module generated by nix";
-            packages = [
-                            pkgs.neurom
-                            ## include all dependencies in the env
-                       ] ++ pkgs.neurom.pythonDeps;
+            packages = with pkgs.pythonPackages; (
+                getPyModRec [ pkgs.neurom ]
+            );
 
             conflicts = conflicts-modules;
+            dependencies = [ python27-full ];
+        };
+
+        neurom-py3 = pkgs.envModuleGen rec {
```

Comment:
JFYI: `neurom` module now brings `ipython` (through `plotly` dependency).
Might be not an issue, but was a bit of a surprise for me.